docs/src/App.js

```diff
@@ -1,10 +1,14 @@
 import './App.css';
```

Comment:
i'm not sure if this is just an issue for me, but i had to add `import React from 'react';` at the top of each `.js` file in order for the application to run. maybe add them just in case?